docs/src/pages/getting-started/installation/dependencies.react.mdx

```diff
@@ -51,3 +51,44 @@ export default function App(props) {
 ```
 </TabItem>
 </Tabs>
+
+If you're planning on using the [Authenticator](/components/authenticator), also wrap your application in `<Authenticator.Provider/>`.
+
+<Tabs>
+<TabItem title="Create React App">
+```jsx
+import { AmplifyProvider, Authenticator } from '@aws-amplify/ui-react';
+import '@aws-amplify/ui-react/styles.css'; // default theme
+
+export default function App() {
+  return (
+    <AmplifyProvider>
+      <Authenticator.Provider>
+        <App />
+      </Authenticator.Provider>
+    </AmplifyProvider>
+  );
+};
+```
+</TabItem>
+<TabItem title="Next.js" >
+```jsx
+import { AmplifyProvider, Authenticator } from '@aws-amplify/ui-react';
+import '@aws-amplify/ui-react/styles.css'; // default theme
+
+export default function App(props) {
+  return (
+    <AmplifyProvider>
+      <Authenticator.Provider>
+        <MyApp {...props} />
+      </Authenticator.Provider>
+    </AmplifyProvider>
+  );
+};
+```
+</TabItem>
+</Tabs>
+
+> The `<AmplifyProvider/>` helps direct [theming](/theming) to all the Amplify components in your application. It can optionally be used with a [theme object](/theming#theme-object), to set the look and feel of your app.
```

Comment:
Unless we are already including angle brackets around component names as a pattern in the docs should we just stick to wrapping in single backticks, e.g. \`AmplifyProvider\`?

Reply:
I did this because earlier in the guide we use it as the component name. But I can change it back here.

Reply:
Back to just the component name 5df98d33ee8b28c55e6af2ff7614e30b0198e5d9